### ISSUE TITLE
Instructor: edit feedback session: show course name and session status in edit form

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#1103]
Showing the session status (together with course ID and name) may be useful.

**Comment by damithc** [comment]
No longer relevant

**Comment by arjunsuri** [comment]
Please guide me on this issue...

**Comment by huai99** [comment]
@damithc  can i work on this issue ? :)

**Comment by whipermr5** [comment]
@huai99 Sure! You are always free to open a PR; we don't require asking :)

**Comment by huai99** [comment]
@damithc  Hi, can you pinpoint to me in the UI where exactly you want to place the course ID and course name? That will be very helpful. Thanks.

**Comment by damithc** [comment]
> @damithc Hi, can you pinpoint to me in the UI where exactly you want to place the course ID and course name? That will be very helpful. Thanks.

We can show the session status here:
![image](https://user-images.githubusercontent.com/1673303/29464719-4c5e3756-8469-11e7-865a-637b815d33b7.png)
The course ID is already there. you can show the course name just below the ID.

**Comment by HirdayGupta** [comment]
@damithc Sir, would this be affected by #7965?

**Comment by damithc** [comment]
> @damithc Sir, would this be affected by #7965?

Yes, I guess so. Once #7965 is merged, this should follow the same.

**Comment by HirdayGupta** [comment]
@damithc Sir, Can I write a PR for this keeping the changes in #7965 in mind, and let it be on hold until it is merged?

**Comment by damithc** [comment]
> @damithc Sir, Can I write a PR for this keeping the changes in #7965 in mind, and let it be on hold until it is merged?

Sure.

**Comment by huai99** [comment]
@HirdayGupta Hi, does that mean that you will be working on this issue ?

**Comment by HirdayGupta** [comment]
Hi @huai99, you are very welcome to work on this Issue if you wish to. You can choose to either wait until #7965 is merged or familiarise yourself with the contents of #7965 and write this Issue's PR accordingly. Since I already know how to do this, I am happy to assist you as well as review your PR to aid your transition into serious contributions for TEAMMATES.

**Comment by huai99** [comment]
@HirdayGupta  Thanks for your help, i will first take some time to familiarize myself with #7965 and check from you if I have any questions. Thanks for being helpful. :)

**Comment by huai99** [comment]
@HirdayGupta  In FeedbackSessionsForm, there is no attribute for the course name, so I will need to add a new attribute for the course name ? Or do you have any suggestions for a simpler approach?

**Comment by nidhi98gupta** [comment]
@damithc As per above comments since #7965 has been merged, can I start working on this? I mean is this issue still on hold?

**Comment by huai99** [comment]
@nidhi98gupta Hi, I have written a PR for this issue and I plan to submit latest by tomorrow, maybe you would like to look at other issues first at the moment? Sorry for the delay
